src/migrations/005_oauth2-redirect-uri.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({id: 5});
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+  await knex('changelog').insert({
+    id: 5,
+    timestamp: Math.floor(Date.now()/1000)
+  });
+
+  await knex.raw(`CREATE TABLE oauth2_redirect_uris (
```

Comment:
@mihok most migrations look like this, and only work with mysql. I wrote a script to generate these files automatically.

Reply:
We'll have to go back through them as I think some of PostgreSQLs migration syntax is slightly different (different types) but this is a good first step having these files generated.